src/app/shared.scss

```diff
@@ -3,32 +3,32 @@ h2 {
 }
 
 .select-box {
-  margin-bottom: 2rem;
-  h3 {
-    font-size: 1.35rem;
-    padding-bottom: 2px;
-    border-bottom: 1px solid #eee;
-  }
-  div {
-    margin: 0 0 0.2rem 0;
-    padding-left: 1.1rem;
-    text-indent: -1.1rem;
-    line-height: 1.4;
-    font-size: 1rem;
+  .mat-checkbox {
```

Comment:
See prior comment/notes, these need to be adjusted to look like the production version